bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,343 @@
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<thing-type id="AirHumidifier">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Air Humidifier via VeSync</label>
+		<description>A Air Humidifier uplinking to VeSync</description>
+
+		<channels>
+			<channel id="enabled" typeId="deviceEnabledType"/>
+			<channel id="display" typeId="deviceDisplayEnabledType"/>
+			<channel id="waterLacking" typeId="deviceWaterLackingType"/>
+			<channel id="humidityHigh" typeId="deviceHighHumidityType"/>
+			<channel id="waterTankLifted" typeId="deviceWaterTankLiftedType"/>
+			<channel id="stopAtTargetLevel" typeId="deviceAutomaticStopReachTargetType"/>
+			<channel id="humidity" typeId="deviceHumidityType"/>
+			<channel id="mistLevel" typeId="deviceMistLevelType"/>
+			<channel id="humidifierMode" typeId="airHumidifierModeType"/>
+			<channel id="nightLightMode" typeId="deviceAFNightLight"/>
+			<channel id="configTargetHumidity" typeId="deviceConfigTargetHumidity"/>
+			<channel id="warmEnabled" typeId="warmModeEnabled"/>
+			<channel id="warmLevel" typeId="warmLevel"/>
+		</channels>
+
+		<properties>
+			<property name="Device Name"/>
+			<property name="Device Type"/>
+			<property name="MAC Id"/>
+		</properties>
+		<representation-property>mac</representation-property>
+
+		<config-description>
+			<parameter name="macId" type="text">
+				<context>text</context>
+				<label>MAC Id</label>
+				<description>The MAC Id of the device as reported by the API.</description>
+			</parameter>
+			<parameter name="deviceName" type="text">
+				<context>text</context>
+				<label>Device Name</label>
+				<description>The name allocated to the device by the app. (Must be unique if used)</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="deviceEnabledType">
+		<item-type>Switch</item-type>
+		<label>Switched On</label>
+		<description>Indicator if the device is switched on</description>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="deviceChildLockEnabledType">
+		<item-type>Switch</item-type>
+		<label>Display Lock</label>
+		<description>Indicator if the devices child lock is enabled (Display Lock)</description>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="deviceDisplayEnabledType">
+		<item-type>Switch</item-type>
+		<label>Display</label>
+		<description>Indicator if the devices display is enabled</description>
+		<state readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="deviceFilterLifePercentageType">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Filter Life Remaining</label>
+		<description>Indicator of the remaining filter life</description>
+		<state readOnly="true" pattern="%.0f %%"/>
+	</channel-type>
+
+	<channel-type id="airPurifierModeType">
+		<item-type>String</item-type>
+		<label>Operation mode</label>
```

Comment:
```suggestion
		<label>Operation Mode</label>
```

Reply:
Going through checking these labels now.

Reply:
I've gone through and updated the case of the label's where I missed capitalisation. :)